Tools/wasm/README.md

```diff
@@ -31,12 +33,18 @@ podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z quay.io/tiran/cpythonbuild:
 docker run --rm -ti -v $(pwd):/python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
 ```
 
+Note that when you exit the container, all work will be lost.
+See the `podman` or `docker` docs for how to preserve the container environment
+across runs.
+
 ### Compile a build Python interpreter
 
+From within the container, run the following commands:
```

Comment:
I see where the instructions can be misunderstood. The instructions assume that all commands are executed from the cpython checkout outside the container **and** from mounted cpython checkout inside the container. Depending on how you set up the container, you may have ``/python-wasm`` as initial working directory. In that case you have to ``cd /python-wasm/cpython`` first.

Reply:
> The instructions assume that all commands are executed from the cpython checkout outside the container [...]

Now I'm confused. The build instructions are to be executed from _inside_ the container, right?